tests/fixtures/AVE-clinical-classification.yaml

```diff
@@ -0,0 +1,35 @@
+type: AssayVariantEffectClinicalClassificationStatement
+subjectVariant:
+  id: ga4gh:VA.t0rDoiIessOWmP0SF0plhXtOwi8TRaZz
+  type: VariationDescriptor
+  variation:
+    id: ga4gh:VA.t0rDoiIessOWmP0SF0plhXtOwi8TRaZz
+    type: Allele
+    location:
+      id: 
+      type: SequenceLocation
+      sequence_id: ga4gh:SQ.w7LxW0PSaVPNsPFvOL24weFjHcfaHyOi
+      interval:
+        type: SequenceInterval
+        start:
+          type: Number
+          value: 10
+        end:
+          type: Number
+          value: 11
+    state:
+      type: LiteralSequenceExpression
+      sequence: T
+  expressions:
+  - type: Expression
+    syntax: hgvs.p
+    value: NP_000305.3:p.Arg11Thr
+    syntax_version: 
+  vrs_ref_allele_seq: R
+predicate: hasAssayVariantEffectFor
+objectAssay: https://github.com/ave-dcd/mave_vocabulary/blob/main/examples/Matreyek_2018.yml
+subjectClassification: BS3_Supporting
+specifiedBy:
+  type: Method
+  subtype: Variant Interpretation Guideline
```

Comment:
This should be a coding but not sure what the `code` would be:

```
  subtype:
    code: HELP
    system: Variant Interpretation Guideline
  reportedIn:
    - type: Document
      pmid: 29785012

```

Reply:
> This should be a coding but not sure what the `code` would be:
> 
> ```
>   subtype:
>     code: HELP
>     system: Variant Interpretation Guideline
>   reportedIn:
>     - type: Document
>       pmid: 29785012
> ```

Not sure what the code should be, beyond the PMID. This is a classification derived from the results of that paper. It could alternatively reference the [Brnich guidelines](https://pubmed.ncbi.nlm.nih.gov/31892348/) but that's just another PMID.

Should we just put a placeholder code here?